pyproject.toml

```diff
@@ -144,5 +144,5 @@ floating-scroll = { version = "==3.2.0", provider = "unpkg", include = ["dist/jq
 "datatables.net-bs4" = { version = "==2.1.8", provider = "unpkg", include = ["css/dataTables.bootstrap4(.min)?.css", "js/dataTables.bootstrap4(.min)?.js"]}
 "datatables.net-buttons" = { version = "==3.2.0", provider = "unpkg", include = ["js/dataTables.buttons(.min)?.js", "js/buttons.colVis(.min)?.js"]}
 "datatables.net-buttons-bs4" = { version = "==3.2.0", provider = "unpkg", include = ["css/buttons.bootstrap4(.min)?.css", "js/buttons.bootstrap4(.min)?.js"]}
-dcmjs = { version = "==0.42.0", provider = "unpkg", include = ["build/dcmjs(.min)?.js(.map)?"] }
+dcmjs = { version = "==0.43.0", provider = "unpkg", include = ["build/dcmjs.min.js(.map)?"] }
```

Comment:
Isn't it useful to also have the non-minified version too for local development? I think that is what we do for most other dependencies.